RevenueCatUI/CustomerCenter/Views/CustomerCenterView.swift

```diff
@@ -112,7 +112,13 @@ private extension CustomerCenterView {
                 WrongPlatformView()
             }
         } else {
-            NoSubscriptionsView(configuration: configuration)
+            if let screen = configuration.screens[.noActive] {
+                ManageSubscriptionsView(screen: screen,
```

Comment:
We probably should now rename the `ManageSubscriptionsView` to something more generic... Can come in a cleanup PR if we're in a hurry though.

Reply:
yes, we have to. I am going to postpone it to prevent conflicts with all the PRs I have opened. But I promise I will do a cleanup on naming 🙏